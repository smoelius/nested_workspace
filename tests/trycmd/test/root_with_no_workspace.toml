args = ["test", "--workspace"]

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/root_with_no_workspace"
