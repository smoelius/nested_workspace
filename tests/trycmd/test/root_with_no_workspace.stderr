...
   Compiling root_with_no_workspace v0.1.0 ([CWD])
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/root_with_no_workspace-[..])
Warning: skipping `[CWD]/empty` as it does not contain a workspace
Warning: found no nested workspaces
